manifests/init.pp

```diff
@@ -737,6 +745,14 @@
     }
   }
 
+  if $::selinux and $manage_selinux_real and $sshd_config_port != '22' {
```

Comment:
We don't want to make assumptions on the initial state of the system. It is better to be declarative and say that we want it and the idempotent nature of Puppet will ensure it is there.